### PR TITLE
Feature/13019 ess u i display enc list in ukho table

### PR DESCRIPTION
**Pull request by Rajesh14881**
Display enc list table with sorting  functionality

### CODE REVIEW

**Comment by crdandoukho** [comment]
@suraj49470 could you please resolve the new merge conflict

File: src/app/features/exchange-set/ess-list-encs/ess-list-encs.component.scss [hunk 1]
```diff
@@ -31,6 +24,13 @@ table th:last-child {
     height: 480px;
     margin: 0px 39px;
 }
+.showSelectedEncWrapper{
```
Comment (crdandoukho) [inline src/app/features/exchange-set/ess-list-encs/ess-list-encs.component.scss:27]: Please delete `showSelectedEncWrapper` class if it's not used

File: src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export class EssUploadFileComponent implements OnInit {
   displayErrorMessage = false;
   validEncList: string[];
   encFile: File;
-  // public displayedColumns = ['Cell name', 'Choose'];
+  private sendErrorMessage: any;
```
Comment (crdandoukho) [inline src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.ts:17]: Please delete `sendErrorMessage` member if not already done when resolving merge conflicts

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Just one CSS class to be deleted please

Also I think sendErrorMessage field in EssUploadFileComponent can be deleted please

**Comment by suraj49470** [comment]
> @suraj49470 could you please resolve the new merge conflict

@crdandoukho  conflict resolved

**Review by crdandoukho — APPROVED** [review]
Looks good. Thanks for your changes

**Comment by Rajesh14881** [comment]
Hi Chris I have done some small align change in showing enc tag, which is showing above table

**Review by crdandoukho — APPROVED** [review]

**Review by rvijayavel — APPROVED** [review]
Looks good

**Pull request merged**